fast_llm/layers/language_model/head.py

```diff
@@ -11,7 +11,13 @@
 from fast_llm.engine.config_utils.tensor_dim import TensorDim, scalar_dim
 from fast_llm.engine.distributed.config import DistributedConfig, DistributedDimNames
 from fast_llm.functional.autograd import grad_is_context, wrap_forward_backward
-from fast_llm.functional.config import CrossEntropyImpl, DistillationLossImpl, TargetFormat, TritonConfig
+from fast_llm.functional.config import (
```

Comment:
Ignoring changes to model head and reverse kl for now

Reply:
ok. cc @oleksost